packages/backend/src/models/GroupsModel.ts

```diff
@@ -221,4 +223,54 @@ export class GroupsModel {
             organizationUuid: updatedGroup.organization_uuid,
         };
     }
+
+    async addUserToGroupsIfExist({
+        userUuid,
+        userId,
+        groups,
+        organizationUuid,
+    }: {
+        userUuid?: string;
+        userId?: number;
+        groups: string[];
+        organizationUuid: string;
+    }) {
+        const organization = await this.database('organizations')
+            .where('organization_uuid', organizationUuid)
+            .first('organization_id');
+        if (!organization) {
+            throw new NotExistsError('Cannot find organization');
+        }
+
+        const existingGroups = await this.database('groups')
+            .whereIn('name', groups)
```

Comment:
Do we hare requirements for case sensitivity?

Reply:
@owlas and I agreed on that - case sensitivity